### ISSUE TITLE
Bring back the “Export variants” functionality into the Nebula simple mode

### ISSUE THREAD

**Issue by tonydisera** [iobio/gene.iobio.vue#699]
From Dennis - I would like to bring back the “Export variants” functionality into the simple mode. However, would it be possible to move the export button from the current location (menu in the upper right corner) into somewhere inside/near the variant panel?

**Comment by AlistairNWard** [comment]
Yeah, let's discuss. The export variants is very much hidden at the moment, so we should move it. I'm not sure it should go in the variants panel, since that could imply exporting variants from the current gene only. Maybe it should go in the left panel?

**Comment by tonydisera** [comment]
@AlistairNWard @mvelinder 

I've changed the icons in the Variants side panel so that we delineate between filters and importing/exporting variants.
<img width="316" alt="Screen Shot 2020-12-15 at 7 51 10 PM" src="https://user-images.githubusercontent.com/8420103/102298813-0f60e600-3f0f-11eb-899b-1a135d3131bd.png">

I also changed the icons on the Genes panel to make these consistent.
<img width="321" alt="Screen Shot 2020-12-15 at 7 53 14 PM" src="https://user-images.githubusercontent.com/8420103/102298872-315a6880-3f0f-11eb-9f1f-7bef78aed768.png">

Do these look okay to you?
